index.html

```diff
@@ -7,7 +7,184 @@
     <link
       rel="stylesheet"
       href="node_modules/bootstrap/dist/css/bootstrap.min.css" />
-    <title>Document</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi"
+      crossorigin="anonymous" />
+    <link rel="stylesheet" href="css/styles.css" />
+    <title>Social Media Page | Home</title>
   </head>
-  <body></body>
+  <body>
+    <!-- Navigation start -->
+    <nav class="navbar">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="#">Social Site</a>
+        <span
+          class="navbar-toggler border-0 p-0"
+          type="button"
+          data-bs-toggle="offcanvas"
+          data-bs-target="#offcanvasNavbar"
+          aria-controls="offcanvasNavbar">
+          <i class="bi bi-list" style="color: #fff; font-size: 2rem"></i>
+        </span>
+        <div
+          class="offcanvas offcanvas-end"
+          tabindex="-1"
+          id="offcanvasNavbar"
+          aria-labelledby="offcanvasNavbarLabel">
+          <div class="offcanvas-header">
+            <h5 class="offcanvas-title" id="offcanvasNavbarLabel">
+              Social Site
+            </h5>
+            <span
+              class="bi bi-x navbar-toggler border-0 p-0"
+              style="color: #fff; font-size: 2rem"
```

Comment:
Do not use inline styles. Make a custom class instead.

Reply:
Replaced inline styles with custom css styles